package-lock.json

```diff
@@ -7042,9 +7042,6 @@
       "version": "1.3.1",
       "resolved": "https://registry.npmjs.org/full-icu/-/full-icu-1.3.1.tgz",
       "integrity": "sha512-VMtK//85QJomhk3cXOCksNwOYaw1KWnYTS37GYGgyf7A3ajdBoPGhaJuJWAH2S2kq8GZeXkdKn+3Mfmgy11cVw==",
-      "requires": {
-        "icu4c-data": "^0.64.2"
-      },
```

Comment:
Hmm ... this change looks un-related to removing Lockwise. Can you remove this particular change from the branch & PR? If not, I'm happy to do it myself.

Reply:
Yes, I can! Sorry about that. Thank you.